Neos.Neos/Classes/Controller/Backend/ModuleController.php

```diff
@@ -93,7 +93,7 @@ public function indexAction(array $module)
 
         $moduleRequest->setArgument('__moduleConfiguration', $moduleConfiguration);
 
-        $moduleResponse = new ActionResponse($this->response);
+        $moduleResponse = new ActionResponse();
```

Comment:
as far as i see it never had a constructor with arguments ...

Reply:
LOL no